Core/Tornado/test/migrationTests/TestClientReturnedValuesAndCredentialsExtraction.py

```diff
@@ -0,0 +1,116 @@
+"""
```

Comment:
I this should go to `DIRAC/tests/Integration` dir.

Reply:
As I was saying in the comments above, I wanted to keep these tests for "memory" for the moment, but I do not intend to run then anymore, even less automatically, so I'd rather keep them here, and remove them eventually.